Jenkins/InedoExtension/JenkinsClient.cs

```diff
@@ -246,8 +233,31 @@ public async Task<JenkinsBuild> GetBuildInfoAsync(string jobName, string buildNu
                 };
             }
         }
+

```

Comment:
Refactored the string concatenation to use consistent helper methods to build the XPath and Urls.